src/porepy/numerics/ad/operators.py

```diff
@@ -1621,16 +1621,31 @@ def set_name(self, name: str) -> None:
     def previous_timestep(self) -> Variable:
         """Return a representation of this variable on the previous time step.
 
-        If the variable is already a representation of the previous time step, the
-        method returns itself.
+        Raises:
+            ValueError:
+                If the variable is a representation of the previous time
```

Comment:
Should this be previous iteration (not time?)

Reply:
Indeed!